tests/test_routes/test_environment_route.py

```
@@ -0,0 +1,244 @@
+    ) as mock_simulate_env:
+        response = client.get("/environments/123/summary")
+        assert response.status_code == 404
+        assert response.json() == {"detail": "Environment not found"}
+        mock_simulate_env.assert_called_once_with("123")
+
+def test_simulate_env_server_error():
+    with patch.object(
+        EnvController,
+        "simulate_env",
+        side_effect=Exception("error"),
+    ) as mock_simulate_env:
```

Ruff

Local variable `mock_simulate_env` is assigned to but never used

(F841)